src/test/java/com/jhuguet/sb_taskv1/app/controller/GiftCertificateControllerTest.java

```diff
@@ -43,13 +43,13 @@ private static void mocksSetuUp() {
     }
 
     @Test
-    void getReturnOne() throws IdNotFound, InvalidIdInputInformation {
-        assertEquals(utils.sampleCertificate().getName(), controller.get("0").get(0).getName());
+    void getReturnOne() throws IdNotFound, InvalidInputInformation {
```

Comment:
Please, describe what your test do, what they check. The name of test should be like 'checkThatOneTagIsFoundIf...'